src/libs/actions/Report.ts

```diff
@@ -4132,92 +4136,113 @@ function markAsManuallyExported(reportID: string, connectionName: ConnectionName
     API.write(WRITE_COMMANDS.MARK_AS_EXPORTED, params, {optimisticData, successData, failureData});
 }
 
+type Params = Record<string, ExportReportCSVParams>;
+
+function exportReportToCSV({reportID, transactionIDList}: ExportReportCSVParams, onDownloadFailed: () => void) {
+    const finalParameters = enhanceParameters(WRITE_COMMANDS.EXPORT_REPORT_TO_CSV, {
+        reportID,
+        transactionIDList,
+    }) as Params;
```

Comment:
I think we don't need `as Params` here

For reference: https://github.com/Expensify/App/blob/2ce48618a1f53952046f33cdb3b81fd9744eb3c6/src/libs/actions/Policy/Category.ts#L1073-L1075